README.md

```diff
@@ -20,46 +26,114 @@ This library requires:
+renders can be a significant time-saving optimization; particularly when the render itself is quick.
+
+If you are using the [default Queue Environment](default-queue-environment), or an equivalent, to run your jobs, then the adaptor will be
+automatically made available to your job. Otherwise, you will need to install the adaptor.
 
 The adaptor can be installed by the standard python packaging mechanisms:
 ```sh
 $ pip install deadline-cloud-for-maya
 ```
 
-After installation it can then be used as a command line tool:
+After installation it can then you can test that it has been installed properly by running the following as the same
+user that runs your jobs:
 ```sh
 $ maya-openjd --help
 ```
 
 For more information on the commands the OpenJD adaptor runtime provides, see [here][openjd-adaptor-runtime-lifecycle].
 
-## Versioning
+### Maya Software Availability in AWS Deadline Cloud Service Managed Fleets
 
-This package's version follows [Semantic Versioning 2.0](https://semver.org/), but is still considered to be in its 
-initial development, thus backwards incompatible versions are denoted by minor version bumps. To help illustrate how
-versions will increment during this initial development stage, they are described below:
+You will need to ensure that the version of Maya that you want to run is available on the worker host when you are using
+AWS Deadline Cloud's [Service Managed Fleets](service-managed-fleets) to run jobs;
+hosts do not have any rendering applications pre-installed. The standard way of accomplishing this is described
+[in the service documentation](https://docs.aws.amazon.com/deadline-cloud/latest/developerguide/provide-applications.html).
+You can find a list of the versions of Maya that are available by default 
+[in the user guide](https://docs.aws.amazon.com/deadline-cloud/latest/userguide/create-queue-environment.html#conda-queue-environment).
 
-1. The MAJOR version is currently 0, indicating initial development. 
-2. The MINOR version is currently incremented when backwards incompatible changes are introduced to the public API. 
-3. The PATCH version is currently incremented when bug fixes or backwards compatible changes are introduced to the public API. 
+## Modifying Jobs Before Submitting
+
+To submit a job, the submitter first generates a [Job Bundle](job-bundle), and then uses functionality from the
+[Deadline](deadline-cloud-client) package to submit the Job Bundle to your render farm to run. If you need to modify
+the job that will be submitted to your farm, then you can use the "Export Bundle" button in the submitter to export the
+Job Bundle to a location of your choice, modify that bundle, and then use the [Deadline Cloud application](deadline-cloud-client)
+to submit that bundle to your farm.
+
+[job-bundle]: https://docs.aws.amazon.com/deadline-cloud/latest/developerguide/build-job-bundle.html
```

Comment:
Is there a customer question that prompted this section? It seems like if someone is using the submitter, it's because they don't want to write a bundle directly.

Reply:
Yes, actually. I know of at least one internal customer that wanted to modify the bundle but wasn't aware of the 'Export bundle' function. The intent of this section is to call attention to that functionality.

Though, I can rephrase this to be more around seeing what will be submitted.